project/Dependencies.scala

```diff
@@ -67,7 +68,7 @@ object Dependencies {
     Orgs.jaegertracing            % "jaeger-client"       % ExampleVersions.jaeger,
     Orgs.jaegertracing            % "jaeger-zipkin"       % ExampleVersions.jaeger,
     Orgs.softwaremillSttpClient3 %% "zio-json"            % ExampleVersions.sttp3,
-    Orgs.d11                     %% "zhttp"               % ExampleVersions.zioHttp,
+    Orgs.d11                     %% "zhttp"               % ExampleVersions.zHttp,
```

Comment:
I do believe we don't need this one anymore since we have just added a `zio-http`.